src/main/java/teammates/common/util/Const.java

```diff
@@ -1396,7 +1396,8 @@ private Const() {
         public static final String STUDENT_UPDATE_PROFILE_NATIONALITY =
                 "Meanwhile, you can provide your nationality "
                 + "<a href=\"" + Const.ActionURIs.STUDENT_PROFILE_PAGE + "\">here</a>.";
-        
+        public static final String DELETED_STUDENT_ATTEMPTING_TO_JOIN =
+                "You can no longer join the course, as you you have been unenrolled from the course by an instructor.";
```

Comment:
Oh yeah, i forgot that when I was testing that, i wasn't logged in as a student, so I thought I did something wrong. Works as expected now.